src/pycrumbs/track.py

```diff
@@ -314,8 +314,6 @@ def write_record(
 
     """
     # Save the record to file
-    if not record_path.name.endswith('.json'):
-        record_path = record_path.with_name(record_path.stem + '.json')
```

Comment:
I see that you have added this functionality in the `tracked` function now, which is fine. But I would prefer to keep it here too (which should not break anything), as this is a public package and this change would represent a backward incompatible change to its behavior (this function is a public function)